src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -6,70 +6,124 @@
 
 import edu.wpi.first.util.sendable.SendableBuilder;
 import frc.robot.RobotConstants;
+import frc.robot.controllers.PositionController;
+import frc.robot.controllers.PositionPreset;
+import frc.robot.controllers.SparkMaxPositionController;
 
 /**
  *
  * @author dcowden
  */
 public class ArmSubsystem extends EntechSubsystem{
 
-  private CANSparkMax elbowMotor;
   private CANSparkMax telescopeMotor;
+  private PositionController controller;
+  
+  public interface Presets {
+	  public static PositionPreset HOME = new PositionPreset("HOME", 0.0);
+	  public static PositionPreset CARRY = new PositionPreset("CARRY", 20.0);
+	  public static PositionPreset SCORE_LOW = new PositionPreset("SCORE_LOW", 20.0);
+	  public static PositionPreset SCORE_MIDDLE = new PositionPreset("SCORE_MIDDLE", 85.0);
+	  public static PositionPreset SCORE_HIGH = new PositionPreset("SCORE_HIGH", 95.0);
+  }
+  public static int TOLERANCE_COUNTS = 50; 
   private boolean enabled = false;
+  private boolean homed = false;
+
   
   public ArmStatus getStatus(){
       return new ArmStatus();
   }
+  public boolean isHome() {
+	  return homed;
+  }
+  
+  public void home() {
+	  controller.setDesiredPosition(Presets.HOME);
+  }
+  
+  public void carry() {
+	  goToPositionIfHomed(Presets.CARRY);
+  }
+  
+  public void scoreLow() {
+	  goToPositionIfHomed(Presets.SCORE_LOW);
+  }
+  
+  public void scoreMiddle() {
+	  goToPositionIfHomed(Presets.SCORE_MIDDLE);
+  }
+  
+  public void scoreHigh() {
+	  goToPositionIfHomed(Presets.SCORE_HIGH);
+  }
   
+  private void goToPositionIfHomed(PositionPreset preset) {
+	  if ( isHome()) {
+		  controller.setDesiredPosition(preset);
+	  }
+	  
+  }
   @Override
   public void initialize() {
 	if ( enabled ) {
-	    elbowMotor = new CANSparkMax(RobotConstants.ARM.ELBOW_MOTOR_ID, MotorType.kBrushed);
 	    telescopeMotor = new CANSparkMax(RobotConstants.ARM.TELESCOPE_MOTOR_ID, MotorType.kBrushed);
```

Comment:
This constant should be RobotConstants.CAN.TELESCOPE_MOTOR_ID.  It needs to change in RobotConstants.java too.